src/maintenance/BatchRepayBadDebtSteward.sol

```diff
@@ -80,9 +79,7 @@ contract BatchRepayBadDebtSteward is IBatchRepayBadDebtSteward, RescuableBase, O
     address[] memory collateralAssets,
     address[] memory users
   ) public override onlyRole(CLEANUP) {
-    uint256 balanceBefore = IERC20(debtAsset).balanceOf(address(this));
```

Comment:
i think there is no need to diff balances, every funds on this contract are here by accident.